tools/tracegen/Port.cpp

```diff
@@ -30,8 +30,10 @@
 #define J9FILE_UNC_EXTENDED_LENGTH_PREFIX (L"\\\\?\\")
 #else /* defined(WIN32) */
 #include <sys/types.h>
-#if !defined(OMRZTPF)
+#if defined(AIXPPC)
 #include <sys/statvfs.h>
+#elif (defined(LINUX) && !defined(J9ZTPF)) || defined(OSX)
```

Comment:
Why are you making this change?  All of the builds were passing.  Did the compiler recently change on the travis builds? Other builds than ran today passed on OSX.